src/twitch/twitch-service.ts

```diff
@@ -64,8 +51,17 @@ const make = Effect.gen(function* (_) {
   Effect.annotateLogs({
     fiber_name: "twitch-service",
   }),
+  Layer.scopedDiscard,
+  Layer.provide(MessagePubSub.Live),
+  Layer.provide(TwitchApiClient.Live),
 );
 
-export const TwitchService = Layer.scopedDiscard(make).pipe(
-  Layer.provide(TwitchServiceRequirementsLive),
-);
+export const TwitchServiceLive = Layer.mergeAll(
```

Comment:
Twitch service now becomes a merge of the currently playing service & the event sub.